.compute

```diff
@@ -18,17 +20,23 @@ data="${SHARED_DIR}/data"
 fis="${data}/LDC/fisher"
 swb="${data}/LDC/LDC97S62/swb"
 lbs="${data}/OpenSLR/LibriSpeech/librivox"
+cv="${data}/mozilla/CommonVoice/en_1087h_2019-06-12/clips"
+npr="${data}/NPR/WAMU/sets/v0.3"
 
 python -u DeepSpeech.py \
-  --train_files "${fis}-train.csv","${swb}-train.csv","${lbs}-train-clean-100.csv","${lbs}-train-clean-360.csv","${lbs}-train-other-500.csv" \
-  --dev_files "${lbs}-dev-clean.csv"\
-  --test_files "${lbs}-test-clean.csv" \
+  --train_files "${npr}/best-train.sdb","${npr}/good-train.sdb","${cv}/train.sdb","${fis}-train.sdb","${swb}-train.sdb","${lbs}-train-clean-100.sdb","${lbs}-train-clean-360.sdb","${lbs}-train-other-500.sdb" \
+  --dev_files "${lbs}-dev-clean.sdb" \
+  --test_files "${lbs}-test-clean.sdb" \
   --train_batch_size 24 \
   --dev_batch_size 48 \
   --test_batch_size 48 \
+  --train_cudnn \
   --n_hidden 2048 \
   --learning_rate 0.0001 \
-  --dropout_rate 0.2 \
-  --epoch 13 \
+  --dropout_rate 0.40 \
+  --epochs 200 \
```

Comment:
Are this many epochs needed? I've found that after about 125-150 it's already converged and a new run, continuing the first, with a lower learning rate is required

Reply:
Will set it to 150 then.